src/java/org/apache/cassandra/index/Index.java

```diff
@@ -136,6 +135,7 @@
  */
 public interface Index
 {
+    public static enum LoadType {READ, WRITE, ALL, NONE};
```

Comment:
+1 to the new enum names. It would be nice to add a very brief comment about what it is, for the sake of the unaware reader. The static and the semicolon are not needed:
```suggestion
    public enum LoadType {READ, WRITE, ALL, NONE}
```

Reply:
Argh! you're SO right! My bad apologies.